### PR TITLE
add Live Activity support

### PR DESCRIPTION
**Pull request by KittenYang**
ref: https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications

### CODE REVIEW

File: APNS/APNS/APNSPusher.swift [hunk 1]
```diff
@@ -15,6 +15,7 @@ public protocol APNSPushable {
                       priority: Int,
                       collapseID: String?,
                       inSandbox sandbox: Bool,
+											liveActivity: Bool,
```
Comment (malayli) [inline APNS/APNS/APNSPusher.swift:18]: @KittenYang Please run `swiftlint autocorrect --config .swiftlint.yml --format` to fix these indentation issues.

File: pusher/pusher.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -11,10 +11,12 @@
 		3A590CE02325E42E005AA8BA /* Application.xib in Resources */ = {isa = PBXBuildFile; fileRef = 3A590CDF2325E42E005AA8BA /* Application.xib */; };
 		3ACF98FD2320E1BC00C40BB4 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3ACF98FC2320E1BC00C40BB4 /* AppDelegate.swift */; };
 		3ACF99012320E1BD00C40BB4 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 3ACF99002320E1BD00C40BB4 /* Assets.xcassets */; };
+		C33DC8BCEF7ABD522D62DFB4 /* Pods_pusher.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 27E8F9EE3A5F9B9E5E773FB1 /* Pods_pusher.framework */; };
```
Comment (malayli) [inline pusher/pusher.xcodeproj/project.pbxproj:14]: @KittenYang Please remove all the modifications made by `pod install`.

File: APNS/APNS/APNSPusher.swift (hunk 1 above)
Comment (KittenYang) [reply APNS/APNS/APNSPusher.swift:18]: Done!

File: pusher/pusher.xcodeproj/project.pbxproj (hunk 2 above)
Comment (KittenYang) [reply pusher/pusher.xcodeproj/project.pbxproj]: @malayli  All Done!

File: APNS/APNS/APNSPusher.swift (hunk 1 above)
Comment (malayli) [reply APNS/APNS/APNSPusher.swift]: Thanks for your work. I'll check on my side to fix this issue.

**Comment by malayli** [comment]
@KittenYang I merged your LiveActivity code to master branch. Thank you for your work.

**Pull request closed without merging**